src/lib.rs

```diff
@@ -217,15 +217,11 @@ impl<T> Stowaway<T> {
             }
             SizeClass::Boxed => Box::into_raw(Box::new(value)),
             SizeClass::Packed => {
-                // If T smaller than *mut T, we need to initialize the extra
-                // bytes. TODO: figure out a way to initialize these bytes (to
-                // the satisfaction of defined behavior) without zeroing them,
-                // if possible.
-                let mut blob: MaybeUninit<*mut T> = if size_of::<T>() < size_of::<*mut T>() {
-                    MaybeUninit::zeroed()
-                } else {
-                    MaybeUninit::uninit()
-                };
+                // If T smaller than *mut T, or contains uninit bytes inernally,
```

Comment:
Propose changing this comment to something like "We need to zero all the bytes, because even if `sizeof(T)` == `sizeof(T*)`, there might still be "unused" and uninitialized padding bytes in the struct